### PR TITLE
[stable10] minimum desktop client version is 2.3.3

### PR DESCRIPTION
**Pull request by patrickjahns**
## Description
We recommend Desktop Client 2.3.3 for ownCloud 10 - with 10.0.10 we intend to bump the min version

## How Has This Been Tested?
🤖 + automated tests

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

**Review by michaelstingl — APPROVED** [review]

**Comment by michaelstingl** [comment]
**ownCloud-2.3.2.4250.pkg**

macos_10_13

**ownCloud-2.2.4.3709.pkg**

macos_10_13

File: apps/dav/tests/unit/Connector/Sabre/BlockLegacyClientPluginTest.php [hunk 1]
```diff
@@ -77,7 +77,7 @@ public function testBeforeHandlerException($userAgent) {
 		$this->config
 			->expects($this->once())
 			->method('getSystemValue')
-			->with('minimum.supported.desktop.version', '2.2.4')
+			->with('minimum.supported.desktop.version', $this->anything())
```
Comment (patrickjahns) [inline apps/dav/tests/unit/Connector/Sabre/BlockLegacyClientPluginTest.php]: Note - we should not care about the input value, as we are not testing that the return value is expected here.

File: apps/dav/tests/unit/Connector/Sabre/BlockLegacyClientPluginTest.php [hunk 2]
```diff
@@ -112,7 +112,7 @@ public function testBeforeHandlerSuccess($userAgent) {
...
```
Comment (patrickjahns) [inline apps/dav/tests/unit/Connector/Sabre/BlockLegacyClientPluginTest.php]: Note - we should not care about the input value, as we are not testing that the return value is expected here.

**Comment by patrickjahns** [comment]
@PVince81 - also just noticed - instead of bumping I made the test less dependent on the actual caller - as it does not matter what version we put in when testing the behavior of it will block lower versions.

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by PVince81** [comment]
setting backport-request label for master port

**Comment by patrickjahns** [comment]
Backport is already here https://github.com/owncloud/core/pull/32658

**Comment by phil-davis** [comment]
Note: in the context that this is the ``stable10`` PR, #32658 is the "forward port"

**Pull request merged**